examples/tutorial/tutorial_notebooks/introduction/exercise_1.ipynb

```diff
@@ -200,6 +214,526 @@
     "df.count()"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Modin on OmniSci\n",
+    "\n",
+    "Let's run examples to run using on `OmniSci`. For this need activate ``Omnisci`` in `StorageFormat` from `modin.config`."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Note: Do not change this code!\n",
+    "import numpy as np\n",
+    "import pandas\n",
+    "import subprocess\n",
+    "import sys\n",
+    "import modin\n",
+    "import modin.config as cfg\n",
+    "cfg.StorageFormat.put('omnisci')"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "metadata": {},
+   "outputs": [
+    {
+     "ename": "AssertionError",
+     "evalue": "Not quite correct. Remember the single line of code change (See above)",
+     "output_type": "error",
+     "traceback": [
+      "\u001b[1;31m---------------------------------------------------------------------------\u001b[0m",
+      "\u001b[1;31mAssertionError\u001b[0m                            Traceback (most recent call last)",
+      "Input \u001b[1;32mIn [2]\u001b[0m, in \u001b[0;36m<module>\u001b[1;34m\u001b[0m\n\u001b[0;32m     16\u001b[0m \u001b[38;5;28;01mexcept\u001b[39;00m:\n\u001b[0;32m     17\u001b[0m     \u001b[38;5;28;01mraise\u001b[39;00m \u001b[38;5;167;01mAssertionError\u001b[39;00m(\u001b[38;5;124m\"\u001b[39m\u001b[38;5;124mDon\u001b[39m\u001b[38;5;124m'\u001b[39m\u001b[38;5;124mt change too much of the original code!\u001b[39m\u001b[38;5;124m\"\u001b[39m)\n\u001b[1;32m---> 18\u001b[0m \u001b[38;5;28;01massert\u001b[39;00m \u001b[38;5;124m\"\u001b[39m\u001b[38;5;124mmodin.pandas\u001b[39m\u001b[38;5;124m\"\u001b[39m \u001b[38;5;129;01min\u001b[39;00m sys\u001b[38;5;241m.\u001b[39mmodules, \u001b[38;5;124m\"\u001b[39m\u001b[38;5;124mNot quite correct. Remember the single line of code change (See above)\u001b[39m\u001b[38;5;124m\"\u001b[39m\n\u001b[0;32m     20\u001b[0m \u001b[38;5;28;01mimport\u001b[39;00m \u001b[38;5;21;01mmodin\u001b[39;00m\u001b[38;5;21;01m.\u001b[39;00m\u001b[38;5;21;01mpandas\u001b[39;00m\n\u001b[0;32m     21\u001b[0m \u001b[38;5;28;01massert\u001b[39;00m pd \u001b[38;5;241m==\u001b[39m modin\u001b[38;5;241m.\u001b[39mpandas, \u001b[38;5;124m\"\u001b[39m\u001b[38;5;124mRemember the single line of code change (See above)\u001b[39m\u001b[38;5;124m\"\u001b[39m\n",
```

Comment:
Did you mean to leave the output? It looks less clean, and it reveals the answer for the user. Better to have the user run the cell and get the AssertionError themself.

Reply:
@jeffreykennethli I deleted not needed outputs.